DataSetExplorer/appsettings.Development.json

```diff
@@ -1,5 +1,15 @@
 {
   "Workspace": {
     "GitClonePath": "../../ClonedProjects/"
+  },
+  "Annotating": {
+    "AvailableCodeSmells": {
+      "Class": [ "Large Class" ],
+      "Method": [ "Long Method" ]
+    },
+    "AvailableHeuristics": {
+      "Class": [ "Class is too long.", "Class is too complex.", "Class has multiple concerns." ],
```

Comment:
I think heuristics should be associated with code smell, not instance type. First read comment I left here https://github.com/Clean-CaDET/platform-explorer-ui-web/pull/6.